src/components/plots/linePlot.ts

```diff
@@ -95,6 +95,30 @@ export module Plot {
       return ["x", "y"];
     }
 
+    protected _getClosestPlotData(queryPoint: Point, datasetKeys: string[], withinValue = Infinity) {
```

Comment:
On normal plot there is a 1to1 mapping between an html element and a datum.  This is not true for line plots